Mathlib/Data/Set/Semiring.lean

```diff
@@ -22,20 +22,19 @@ open Pointwise
 
 variable {α β : Type*}
 
--- Porting note: mathlib3 uses `deriving Inhabited, PartialOrder, OrderBot`
```

Comment:
Let's rather turn this into a TODO, and point at https://github.com/leanprover-community/mathlib4/issues/380
```suggestion
-- The following instances should be constructed by a deriving handler.
-- https://github.com/leanprover-community/mathlib4/issues/380
```